### DISCUSSION TITLE
目前支持的镜像列表

### DISCUSSION THREAD

**Discussion by cuisongliu** [labring/sealos#1198] [Announcements]
目前已经同步到hub.docker

https://hub.docker.com/u/labring 所有能看到的版本都是可用的

**Comment by 0x01toby** [comment]
感谢分享，这些应该是构建sealos底层的核心基础组件：
- 计算基础组件：k8s
- 网络基础组件：calico
- 软件管理基础组件: helm
- 数据库基础组件: mysql-oprator
- 缓存中间件基础组件: redis-oprator
- 网关基础组件: ingress-nginx
- 块存储/文件基础组件: openebs
- 对象存储基础组件: minio-oprator
- DNS基础组件: coredns
这些组件的选取目前有相关的规范吗？

**Reply by cuisongliu** [reply]
这些镜像都维护在 https://github.com/labring/cluster-image/tree/main/runtime  这里了，coredns主要是一个配置的集群镜像，这个为了解决有的服务器上没有DNS的问题，比较优雅的方案。如果有更好的镜像可以提交到这个仓库

**Comment by cuisongliu** [comment]
https://github.com/labring/cluster-image/tree/main/docs  具体文档在这里

**Comment by katelynn620** [comment]
@cuisongliu 不好意思借串問一下，目前在嘗試部署私有化Sealos，但`docker-terminal`這個鏡像似乎沒有公開存取，想請教一下這是以後會開放嗎？還是沒有打算開放呢？謝謝

![image](https://github.com/labring/sealos/assets/816515/28dd5049-dbfc-44fa-ae8d-4c02392572ac)

**Reply by bxy4543** [reply]
[terminal image repo change to](https://github.com/labring/sealos/pull/3836/files) `docker.io/labring4docker/terminal:1.20.4-6`
